packages/puppeteer-core/src/api/Browser.ts

```diff
@@ -24,6 +24,7 @@ import {
   timeout,
   fromAbortSignal,
 } from '../common/util.js';
+import type {Cookie} from '../puppeteer-core.js';
```

Comment:
Nit: wrong import, mostly likely due to IDE auto-import.

Reply:
Good catch! Fixed.